### ISSUE TITLE
New install using main branch, can you downgrade to older slurm?

### ISSUE THREAD

**Issue by wccropper** [clustervision/trinityX#445]
We just installed using the main branch, and would like to downgrade to an older slurm if possible. This is to ease migration from existing cluster. We are running slurm-22.05.11.

**Comment by aphmschonewille** [comment]
yes, i think it'd work. as long as you no longer run ansible and accidentally upgrade slurm again, or at least skip tags for slurm.
